CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 2.0.0 (07/19/2021)
+
+* __breaking__ Drop support for Python 3.5 (thanks [@nebnola](https://github.com/nebnola))
```

Comment:
I missed that PR and I don't really understand why we did that. Isn't Pyton 3.5 relatively new?

Reply:
Python 3.5.0 was released 09/13/2015 and reached end-of-life in 09/20/2020. Assuming you're not running a very old LTS release (e.g. Ubuntu 16.04), you should be running minimally Python 3.6+.

The move to 3.6 does allow rewriting our string handling to use f-strings which is a nice boon in my mind, and again, no reason to hold ourselves back for the sake of EOL software.

Reply:
Oh, thanks.